core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -156,6 +156,10 @@ enum class UniqueType(
     LessPolicyCostFromCities("Each city founded increases culture cost of policies [relativeAmount]% less than normal", UniqueTarget.Global),
     LessPolicyCost("[relativeAmount]% Culture cost of adopting new Policies", UniqueTarget.Global),
 
+    /// Tech Cost
+    LessTechCostFromCities("Each city founded increases science cost of technologies [relativeAmount]% less than normal", UniqueTarget.Global),
```

Comment:
```suggestion
    LessTechCostFromCities("Each city founded increases Science cost of Technologies [relativeAmount]% less than normal", UniqueTarget.Global),
```
Let's not big ourselves into a deeper hole ;)